invenio/base/templates/page_base.html

```diff
@@ -153,6 +153,7 @@
   {% endblock %}
 {%- endblock %}
 {%- block javascript %}{% endblock %}
+{%- block trackingcode %}{% endblock %}
```

Comment:
The issue I have seen is that many templates in modules use the ``javascript`` block like https://github.com/inveniosoftware/invenio/blob/pu/invenio/modules/deposit/templates/deposit/run_base.html#L67

So if in the overlay template you don't want to use the ``javascript`` part and remove that block/rewrite it you will lose as well the tracking code (which is not desired)

Like this, we can separate the tracking code (see https://github.com/jmartinm/inspire-next/blob/6337772406d9aa40134d4697ccc88a26899bc465/inspire/base/templates/page.html#L50) and don't conflict with ``javascript`` block.